### PR TITLE
Support schemaless resources for _fields (ref #384)

### PR DESCRIPTION
**Pull request by leplatrem**
@MrChoclate @Natim review?

### CODE REVIEW

File: cliquet/tests/resource/test_views.py [hunk 1]
```diff
@@ -704,3 +704,33 @@ def test_next_page_url_relies_on_headers_information(self):
...
+        result = resp.json['data'][0]
+        self.assertNotIn('owner', result)
+
+    def test_specified_fields_are_included(self):
+        resp = self.app.get(self.collection_url + '?_fields=size,category')
+        result = resp.json['data'][0]
+        self.assertIn('size', result)
+        self.assertIn('category', result)
+
+    def test_unknown_fields_are_ignored(self):
```
Comment (MrChoclate) [inline cliquet/tests/resource/test_views.py]: Is this for schemaless? I think so but cannot find where it is said.

**Comment by MrChoclate** [comment]
LGTM

**Comment by Natim** [comment]
I think we should always have ``id`` and ``last_modified`` even if they are not in ``_fields`` because they are part of the protocol.

**Comment by leplatrem** [comment]
I merge like this and will take @Natim's into account in another PR

**Pull request merged**